contrib/depends/packages/rustc.mk

```diff
@@ -11,7 +11,16 @@ endef
 define $(package)_config_cmds
 endef
 
+# Remove blobs from source
+# TODO: script here could be less messy
+
 define $(package)_preprocess_cmds
+  rm -rf src/llvm-project && \
+  find . -type f -regex ".*\.\(a\|dll\|exe\|lib\)$$$$" -delete && \
+  find . -type f -name ".cargo-checksum.json" -print0 | xargs -0 -I% sh -c 'echo "{\"files\":{}}" > "%"' && \
+  find . -type f -name "Cargo.lock" -delete && \
```

Comment:
Removing the `Cargo.lock` file is necessary because we're removing binaries from the **source** code, causing checksum mismatches.

Builds are still reproducible. All dependencies are vendored, and we don't have network access here. Version numbers are included in `Cargo.toml`. 

This PR isn't ready for review.

Reply:
Sorry, I didn't mean to criticize a WIP. I just wanted clarity, or to point out an error which would presumably make your life more difficult. I now have the former.

My one other chime in is the version numbers in the toml aren't exact, yet ranges, but sounds like that won't be an issue.